File: Calculadora/Velocidade Média.cpp
#include<bits/stdc++.h>

using namespace std;

void resultado1();

int main(){
    double dUm, dDois, tempoUm, tempoDois, resultado;

    cout << "Posição Inicial: ";
    cin >> dUm;

    cout << "Posição Final: ";
    cin >> dDois;

    cout << "Tempo Inicial: ";
    cin >> tempoUm;

    cout << "Tempo FInal: ";
    cin >> tempoDois;

    resultado1();
}

void resultado1(){
    double resultado, dDois, dUm, tempoUm, tempoDois;
    resultado = (dDois - dUm) / (tempoDois - tempoUm);

    cout << resultado;
};

File: algorithms/Calculadora/Velocidade Média.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Velocidade_Média
{
    class Program
    {
        static void Main(string[] args)
        {
            double dUm, dDois, tempoUm, tempoDois;

            Console.Write("Posição Inicial: ");
            dUm = double.Parse(Console.ReadLine());

            Console.Write("Posição Final: ");
            dDois = double.Parse(Console.ReadLine());

            Console.WriteLine();

            Console.Write("Tempo Inicial: ");
            tempoUm = double.Parse(Console.ReadLine());

            Console.Write("Tempo Final: ");
            tempoDois = double.Parse(Console.ReadLine());

            Console.WriteLine();

            Console.WriteLine(Velocidade(dUm: dUm, dDois: dDois, tempoUm: tempoUm, tempoDois: tempoDois));
            Console.ReadLine();
        }

        public static double Velocidade(double dUm, double dDois, double tempoUm, double tempoDois)
        {
            double resultado = (dDois - dUm) / (tempoDois - tempoUm);
            return resultado;
        }
    }
}
